.github/workflows/integrate.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           provider: microk8s
           channel: ${{ matrix.microk8s-versions }}
           microk8s-addons: "dns hostpath-storage rbac metallb:10.64.140.43-10.64.140.49"
-          juju-channel: 3.1/stable
+          juju-channel: 3/stable
```

Comment:
Yeah, I was debating this one. The benefit of not pinning it is to keep it up to date as time goes by, but I guess that would be better in `main` than in the track branch, this is just to guarantee that at this point in time this charm was tested with this version of juju. On the other hand I can see us bumping this again (for 3.5 for instance), so I wonder if it's really worth pinning.

Reply:
I think you're right about how we'll bump it soon, but imo your other PR #405 is a good example of why we shouldn't pin too broadly since in future it might not work :D

Reply:
+1

Reply:
Updated @ca-scribner let me know if this resolves the concern. We can think of better approaches for making these bumps easier and faster to keep up to date with juju and other deps.